contracts/MapleAddressRegistryETH.sol

```diff
@@ -226,4 +226,14 @@ contract MapleAddressRegistryETH {
     address constant public syrupRecapitalizationModule = 0x5dfe0460f66fa06bFCbB3211e723556be6B3f69D;
     address constant public syrupStSyrup                = 0xc7E8b36E0766D9B04c93De68A9D47dD11f260B45;
 
+    /**************************************************************************************************************************************/
+    /*** ExternalContracts                                                                                                              ***/
```

Comment:
Two words, needs a space

Reply:
Would prefer external protocol contracts, external contracts alone is too general, like oracles and USDC would fall in that for example.